applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -192,18 +199,31 @@ SETTINGS_STATIC_HANDLER_DEFINE(ble_bond, MODULE_NAME, NULL, settings_set, NULL,
 static uint8_t get_bt_stack_peer_id(uint8_t id)
 {
 	if (IS_ENABLED(CONFIG_BT_PERIPHERAL)) {
-		if ((state == STATE_ERASE_PEER) ||
-		    (state == STATE_ERASE_ADV)) {
+		if ((state == STATE_ERASE_PEER) || (state == STATE_ERASE_ADV) ||
```

Comment:
Maybe we could add a checks for groups of states as small inline functions? I think, that could make code easier to understand and less bug prone.

Reply:
yes agree that code is not clear.  However in most we check for different state groups so doing it in separate function wouldn't solve a problem

Reply:
Discussed offline